.github/workflows/pythonbuild.yml

```diff
@@ -259,9 +259,12 @@ jobs:
           uv pip freeze
       - name: Install FlyteCTL
         uses: unionai-oss/flytectl-setup-action@master
+        with:
+          # Remove version and go back to latest when unionai-oss/flytectl-setup-action is fixed
+          version: "0.8.18"
       - name: Setup Flyte Sandbox
         run: |
-          flytectl demo start
+          flytectl demo start --version=v0.8.18
```

Comment:
we don't need to specify the version here. This flag maps to a flyte release, not flytectl.

Reply:
ditto.